package.json

```diff
@@ -142,7 +141,8 @@
   "resolutions": {
     "@types/react": "16",
     "@types/react-dom": "16",
+    "aria-query": "5.3.0",
```

Comment:
**suggestion:** What do you think about adding a comment explaining why we had to pin this version? (similar to `strip-ansi` below).

Reply:
Oh yeah, honestly, we probably don't even need this change, it turned out to not matter. I'll remove it.